### ISSUE TITLE
German Translation for Cross Cutting Concerns

### ISSUE THREAD

**Issue by alxlo** [isaqb-org/curriculum-foundation#656]
Pull request https://github.com/isaqb-org/curriculum-foundation/pull/649 brought to light (again) that we have some trouble to agree on a good German translation for the term Cross Cutting Concerns as used in LZ 03-10 and LZ 04-07.

In short: "Querschnittsthemen" is in @alxlo 's opinion overly broad and general. The translation I'm used to and that is regularly used in scientific literature is "Querschnittsbelange".

Thema" refers to a general topic or subject, while "concern" in software architecture implies a specific area of interest or responsibility within the system. "Belang" more accurately captures this sense of importance and relevance.

**Comment by rhoadesre** [comment]
German is not my mother tongue, but "Querschnittsbelange" sounds really strange for me. 

Are there any references to that term in the literature? "Querschnittsthemen" do exist in the literature.

I will definitely let the Germans decide this one.   :-)

**Comment by alxlo** [comment]
https://dict.leo.org/englisch-deutsch/cross-cutting%20concern

https://de.wikipedia.org/wiki/Cross-Cutting_Concern

> Cross-Cutting Concern (CCC) ist ein Begriff der [Informatik](https://de.wikipedia.org/wiki/Informatik), der im Kontext des [Teile-und-Herrsche-Prinzips](https://de.wikipedia.org/wiki/Teile_und_herrsche_(Informatik)) so genannte querschnittliche Belange einer Software bezeichnet,...

https://www.google.com/search?q=%22querschnittliche+Belange%22 and https://www.google.com/search?q=querschnittsbelang&nfpr=1&sa=X&dpr=1.2 provide quite a large amount of suitable hits.

We also used that term already in our glossary https://github.com/isaqb-org/glossary/blob/main/docs/1-terms/C/term-cross-cutting-concern.adoc (and as far as I interpret the commit history, I didn't meddle with this in the more recent past.)

I'm not a huge fan of that term as it is not that common but nevertheless way better than "Thema" IMHO
https://www.dwds.de/wb/Belang

**Comment by rhoadesre** [comment]
@alxlo: 
1. Adding a reference to "leo" is definitely not valid because the term is taken out of context
2. I was; however, surprised at how many references there are from your google search etc.
I'm fine either way.

**Comment by ulibecker** [comment]
Yes, "Thema" in isolation is a very broad term, but we use it in conjunction with "Querschnitt" and that is fine for me. I know the term "Belang" mainly from the university context (e.g., from aspect-oriented software development). I am fine with both "Querschnittsthemen" and "Querschnittsbelange".

**Comment by gernotstarke** [comment]
wir verwenden sowohl Querschnittsthemen als auch Querschnittskonzepte
